scripts/get_version.mjs

```diff
@@ -44,6 +44,8 @@ export async function getVersion(platform) {
       return "1.7.0";
     case "linux":
       return "1.4.0";
+    case "test":
```

Comment:
IDK how you feel about this. Maybe we just skip the `environment_json` when platform is test

Reply:
Actually, shouldn't `platform` be `jasmine`?

Reply:
Test is not a platform. You could run tests on many different platforms.

I guess the problem is more nuanced that we are considering.
The app platform/OS can be seen separate from the web app platform.
The web app platform is really Cordova and Electron. And now Jasmine.

However, some of the code will not actually run on Jasmine, because they don't have the Cordova or Electron APIs available (e.g. cordova.{platformId,exec}, ipcRenderer)

I think currently all the tests run on jasmine, but we need to figure out how to run cordova or electron tests in the future.

Going back to get_version, that's really for the app, not the tests. We shouldn't be calling it.

Reply:
Yeah, alright. I'll make a separate action then.

Reply:
Calling webpack directly in the test action will fix that, thanks for the suggestion.